product-manual/koala-agents.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "AI Agents"
+description: "Build a custom AI agent to automate your sales workflows"
+---
+
+## What is an agent?
+
+Agents are a way to automate your sales workflows. You can use them to simply answer questions about a profile or account, or to qualify leads and accounts.
+
+There are a few ways of creating an agent:
+- When setting up qualification in a play
+- AI agents on settings
+- Research agents
+
+Let's dive into each of these.
+
+## Qualification agents
+
+You can use a qualification agent to automatically qualify leads and accounts based on research. 
+
+### Creating an agent
+
+To create an agent you can use for qualification, select or create a new agent in the third step of the `Create New Play` flow:
+
+![Qualification agent](/images/product-manual/koala-agents/qualification.png)
+
+To create an AI agent, you'll need to:
+
+1. **Agent goal** - Define the purpose of your agent (required)
+2. **Question/Prompt** - Ask the specific question you want answered (required)
+3. **Research type** - Choose between "Companies" or "People" (required)
+4. **Research individual people** - Option for people-focused research
+5. **Response Type** - Select how the AI should format its answer
+6. **Advanced Options** - Additional configuration settings
+
+![Qualification agent](/images/product-manual/koala-agents/creating-ai-agent.png)
+
+For qualification agents, the **Response Type** is particularly important. Based on those you'll set up criteria for qualifying leads and accounts.
```

Comment:
This is probably worth using a Note callout component.

I'd also highly recommend using boolean, number or picklist response types because these will be easier to create qualification conditions with.